help/en/html/tools/logixng/LogixNG.shtml

```diff
@@ -17,72 +16,151 @@
+      <i>ConditionalNG</i> is the same as a <i>Conditional</i>.
+      An <i>expression</i> in LogixNG is similar to a
+      <i>conditional variable</i> in Logix and an <i>action</i> in
+      LogixNG is similar to a <i>conditional action</i> in Logix.</p>
+
+      <p>LogixNG also has references and tables. A reference is an
+      indirect pointer to an item on the layout. For example, if
+      the memory IM1 has the value "IT1", the reference {IM1} points
+      to the turnout IT1. This means that if an action has the
+      command "set turnout {IM1} to thrown", it means that LogixNG
+      reads the memory IM1, which in this example has the value IT1,
+      and then sets the turnout IT1 to thrown.  This is similar to the Logix
+      Indirect using <strong>@memoryName</strong>.</p>
+
+      <p>References can be used to change which turnout to use, which
+      is very useful to setup generic rules. If a particular rule is
+      common for many places on the layout, references can be use to
+      use that single rule for all these cases.</p>
+
+      <p>LogixNG also adds tables to JMRI. A table is similar to
+      Memory but has one or two dimensions. Each row and each column
+      in the table may be given a name, which means that a single
+      cell in the table can be referred to with the row name and the
+      column name. Tables are in particular intended to be used
+      together with references, to make it easy to setup complex
+      configurations. A table can be created in a spreadsheet software
+      like Microsoft Excel or LibreOffice Calc and then exported as an
+      CSV file and imported to JMRI.</p>
 
-<!--
-	  <p>LogixNG, short for Logix Next Generation, provide a
-	  powerful capability for JMRI to monitor one or more
-	  conditions on a layout, and take action when these
-	  conditions change in a user-specified way. LogixNG can be
-	  used to control signals, crossing gates, and other types of
-	  automation on the layout. The user interface is designed to
-	  be user friendly to all users with basic familiarity with
-	  JMRI. A LogixNG provides a means for setting up
-	  user-specified logic in an intuitive manner, without the
-      user having to be familiar with mathematical logic.</p>
-
-      <p>LogixNG is similar to Logix&trade;, but is a new design
-	  to make the code easier to manage and extend.</p>
--->
-
-
-	  <a name="contents" id="contents"></a>
+      <ul>
+        <li><a href="#components">LogixNG Components</a></li>
+        <li><a href="#preferences">LogixNG Preferences</a></li>
+      </ul>
+
+      <a name="components" id="components"></a>
+      <h2>LogixNG Components</h2>
+
+        <p>The LogixNG tool has three main components.  They are accessed via the main PanelPro
+        <strong>Tools &rArr; Tables &rArr; LogixNG</strong> menu path like other JMRI table items.
+        The 3 LogixNG tables and related content will be <strong><em>loaded</em></strong>
```

Comment:
I recommend changing this to:
`The three LogixNG tables and ...`

Reply:
One of the issues is that LogixNG is both a set of tools that are inter-related and the primary component.  Is "LogixNG" a tool or a bean?  The answer is yes but it depends on the context.  I was trying to clarify the distinction.

I will revisit this in phase 2.  I hate to give up clean CI runs.